ts/src/binance.ts

```diff
@@ -4967,7 +4967,11 @@ export default class binance extends Exchange {
                 'Refund Failed': 'failed',
             },
         };
-        const statuses = this.safeValue (statusesByType, type, {});
+        const statuses = this.safeValue (statusesByType, type, {
```

Comment:
@pcriadoperez when will we fallback to the default value? isn't easier to add a new category to `statusByType`?

Reply:
You are right. I changed to add the type in withdrawals and deposit, so then it also gets the correct status type, I think it's a cleaner solution to what I had before.